GameDataParser/Parsers/ExpParser.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.MemoryMappedFiles;
+using System.Xml;
+using GameDataParser.Crypto.Common;
+using GameDataParser.Files;
+using Maple2Storage.Types.Metadata;
+using ProtoBuf;
+
+namespace GameDataParser.Parsers
+{
+    class ExpParser
+    {
+        public static List<ExpMetadata> Parse(MemoryMappedFile m2dFile, IEnumerable<PackFileEntry> entries)
+        {
+            List<ExpMetadata> expList = new List<ExpMetadata>();
+            foreach (PackFileEntry entry in entries)
+            {
+
+                if (!entry.Name.StartsWith("table/nextexp"))
+                {
+                    continue;
+                }
+
+                using XmlReader reader = m2dFile.GetReader(entry.FileHeader);
```

Comment:
Does the `using` keyword do anything here?

Reply:
I'm not sure, copied from other classes. I don't think it matters, maybe @rhydev can answer what it does.

Reply:
I tested with and without and didn't change the outcome. Will leave this conversation to future discussion.